P-GPS data in external flash (#9212)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * :ref:`lib_lwm2m_location_assistance` library that has support for using A-GPS, P-GPS and ground fix assistance from nRF Cloud using an LwM2M server.

* :ref:`lib_nrf_cloud_pgps` library:

  * Added:

    * Added access to P-GPS predictions in external flash.

  * Fixed:

    * An issue where 0 predictions would be requested from the cloud.

>     * An issue when not using a custom download transport, where subsequent updates were locked out after first one completes.

Reviewer: ```suggestion
    * An issue, where subsequent updates were locked out after the first one completes.
      This happened when custom download transport was not used.
```

  * :ref:`lib_lwm2m_location_assistance` library that has support for using A-GPS, P-GPS and ground fix assistance from nRF Cloud using an LwM2M server.

* :ref:`lib_nrf_cloud_pgps` library:

  * Added:

    * Added access to P-GPS predictions in external flash.

  * Fixed:

    * An issue where 0 predictions would be requested from the cloud.

>     * An issue, where subsequent updates were locked out after the first one completes.

Author: Not to me, but that's how @peknis worded it in his change request above.

>     * An issue, where subsequent updates were locked out after the first one completes.

Reviewer: My bad. Had it there by mishap. No comma needed
